src/components/org/apache/jmeter/visualizers/TableVisualizer.java

```diff
@@ -205,6 +226,26 @@ public void run() {
             }
         });
     }
+    
+    /**
+     * When the limit is reached, JMeter will not {@link SampleResult} anymore
+     * Setting to 0 disables the protection
```

Comment:
Is "Setting to 0 disables the protection" relevant here?
`@return ...` would be good